src/com/jgranados/koky/ui/KokyFrame.java

```diff
@@ -59,7 +59,7 @@ public KokyFrame() {
         languageLabel.setText(super.language.name());
         txtInstruction.requestFocusInWindow();
         this.getContentPane().setBackground(new java.awt.Color(0, 153, 0));
-        this.saveFileChooser.setFileFilter(new FileNameExtensionFilter(KOK_EXTENSION_DESC, KOK_EXTENSION));
+        this.saveFileChooser.setFileFilter(new FileNameExtensionFilter(KOK_EXTENSION_DESC, KOK_EXTENSION, JPG_FILE_EXTENSION));
```

Comment:
Podrias cambiar el valor de KOK_EXTENSION_DESC para que describa algo asi como "Archivos Kok o JPG"

Reply:
¡Buena observación!

Reply:
Hola, te comento que me esta dando el siguiente mensaje al momento de tratar de subir el cambio:

remote: Resolving deltas: 100% (4/4), completed with 4 local objects.
remote: error: GH006: Protected branch update failed for refs/heads/feature-anex-02.
remote: error: At least 1 approving review is required by reviewers with write access.
To https://github.com/jgranadosiastic/Koky.git
 ! [remote rejected] feature-anex-02 -> feature-anex-02 (protected branch hook declined)
error: failed to push some refs to 'https://github.com/jgranadosiastic/Koky.git'

El branch me aparece protegido.

Reply:
Hola, creo que le logré quitar la protección. Prueba de nuevo por favor.

Reply:
Fijate que me sigue dando el mismo mensaje :(

remote: Resolving deltas: 100% (9/9), completed with 6 local objects.
remote: error: GH006: Protected branch update failed for refs/heads/feature-anex-02.
remote: error: At least 1 approving review is required by reviewers with write access.
To https://github.com/jgranadosiastic/Koky.git
 ! [remote rejected] feature-anex-02 -> feature-anex-02 (protected branch hook declined)
error: failed to push some refs to 'https://github.com/jgranadosiastic/Koky.git'

Reply:
Ahorita ya le quité toda restriccion. intenta otra vez porfa

Reply:
Listo